helpers_test.go

```diff
@@ -223,11 +223,38 @@ func TestGetCassandraType(t *testing.T) {
 				Elem:       NativeType{typ: TypeDuration},
 			},
 		},
+		{
+			"vector<float, 3>", VectorType{
+				NativeType: NativeType{
+					typ:    TypeCustom,
+					custom: VECTOR_TYPE,
+				},
+				SubType:    NativeType{typ: TypeFloat},
+				Dimensions: 3,
+			},
+		},
+		{
+			"vector<vector<float, 3>, 5>", VectorType{
```

Comment:
It would be nice to have a couple of cases with more complex types like `vector<map<uuid,timestamp>, 5>` for example